tests/unit/runtime/pipe/test_pipe.py

```diff
@@ -41,9 +41,11 @@ class TestPipeCifar10(DistributedTest):
     def test(self, topo_config):
         skip_on_arch(min_arch=7)
 
+        global_batch_size = 4
+        assert global_batch_size % self.world_size == 0
         config_dict = {
-            "train_batch_size": 16,
-            "train_micro_batch_size_per_gpu": 4,
+            "train_batch_size": global_batch_size,
+            "train_micro_batch_size_per_gpu": int(global_batch_size / self.world_size),
```

Comment:
It is much better to set `gradient_accumulation_steps` here to 1 rather than setting `micro_batch_size`